app/resources/v1/__tests__/users_pg.test.js

```diff
@@ -0,0 +1,126 @@
+/* eslint-env jest */
+import request from 'supertest'
+import { setupMockServer } from '../../../../test/helpers/setup-mock-server'
+import users from '../users_pg'
+
+jest.mock('twitter')
+jest.mock('../../../db/models', () => {
```

Comment:
Same comments in this file re:
1. mocks
2. consts

Also, this file should keep the `users.test.js` filename (no `_pg` suffix).